packages/lib/docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,52 @@
+= Upgrades
+
+*Develop upgradeable smart contracts securely.*
```

Comment:
```suggestion
*Deploy and upgrade smart contracts securely* from your JavaScript code.

This library powers the xref:cli::index.adoc[*OpenZeppelin CLI*], by implementing all its deployment and upgrade operations. You can call into this library directly from your scripts if you do not want to go through the CLI. 

WARNING: The upgrades library only creates and upgrades contracts. It will not keep the xref:cli::configuration.adoc[CLI configuration] files up to date with new deployments, meaning that you will have to track your contract addresses on your own. We will be working on a new version of this library that can handle this in the future.
```